webhook/resourcesemantics/defaulting/defaulting.go

```diff
@@ -172,6 +173,12 @@ func (ac *reconciler) reconcileMutatingWebhook(ctx context.Context, caCert []byt
 			continue
 		}
 		webhook.Webhooks[i].Rules = rules
+		webhook.Webhooks[i].NamespaceSelector = &metav1.LabelSelector{
+			MatchExpressions: []metav1.LabelSelectorRequirement{{
+				Key:      "pkg.knative.dev/skipWebhooks",
```

Comment:
Think k8s labels are normally hyphen-case, so this may be better as:

```suggestion
				Key:      "pkg.knative.dev/skip-webhooks",
```

Reply:
I'm looking for feedback on what we should name it, if we want to name it something different. I'll update with the hyphen for now.

Reply:
ah ok - I didn’t realise there’s already a conversation on the name.. I’ll let others bike-shed on that, as long as it’s hyphen-case 😅